functions/src/modules/Users.functions.ts

```diff
@@ -0,0 +1,89 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import GoogleSheet from '../services/GoogleSheets';
+import { DataSnapshot } from 'firebase-functions/lib/providers/database';
```

Comment:
I'm not sure this looks fine… Do we really need to import deeply from inside the library? What is the benefit and is it a best practice?